docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Notifications
+description: Learn about different push notification services.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo apps can work with any notification service or capabilities offered by the operating system thanks to [config plugins](/config-plugins/introduction/). The following services provide purpose-built Expo integrations for implementing push notifications in your app:
+
+> The [`expo-notifications`](/versions/latest/sdk/notifications/) library is designed and tested to work with Expo's push notification service. Some advanced features may not be compatible with third-party providers, as they often have their own native and React Native SDKs optimized for their services.
+
+### Expo Notifications
+
+[Expo Notifications](/push-notifications/overview) provides a unified API for handling push notifications across Android and iOS. It integrates seamlessly with your Expo account and is free to use.
```

Comment:
```suggestion
[Expo Notifications](/versions/latest/sdk/notifications/) provides a unified API for handling push notifications across Android and iOS. It integrates seamlessly with your Expo account and is free to use.
```

The link goes to the Push notifications service page instead of the library where as in the content we referencing the library.

Reply:
Maybe you want to change it to Expo push notifications?